scripts/task-central.sh

```
@@ -0,0 +1,33 @@
+    echo "  ./scripts/task-central.sh list"
+    echo "  ./scripts/task-central.sh show 11.1"
+    echo "  ./scripts/task-central.sh set-status --id=11.1 --status=done"
+    echo "  ./scripts/task-central.sh next"
+    echo ""
+    echo "📍 Current worktree: $(pwd)"
+    echo "📍 Main worktree: $MAIN_WORKTREE"
+    exit 0
+fi
+
+# Run the command in main worktree
+echo "🔄 Running in main worktree: task-master $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)